README

```diff
@@ -274,3 +274,68 @@ refresh			Attempts to reinsert all modules that are currently
 xml			Build a policy.xml from the XML included with the
 			base policy headers and any XML in the modules in
 			the current directory.
+
+
+## How to work with SELinux
+### Introduction
+
+To be able to add and change the SELinux policy, you need to clone the repository with refpolicy, and make changes with a separate commit.
```

Comment:
Please write instructions in indirect form(withou you)
e.g. 
To add and change the SELinux policy, the repository with refpolicy should be cloned and changes are done with a separate commit.

Reply:
Done